### ISSUE TITLE
Dropdown menus are not accessible via keyboard navigation

### ISSUE THREAD

**Issue by SoftVision-CiprianMuresan** [mozilla-extensions/normandy-devtools#522]
**[Affected versions]:**
- Firefox Nightly 83.0a1, Build ID 20201014214248

**[Affected Platforms]:**
- Windows 10

**[Prerequisites]:**
- Have VPN access to the staging environment.
- Have the NDT addon (version 2.1.1) installed on a new profile.

**[Steps to reproduce]:**
1. Open the Firefox profile from prerequisites and click the NDT button from the Browser Toolbar.
2. Switch to the `Stage` environment and click the `Create Recipe` button.
3. Focus the `Experimenter Slug` input field and press the `Tab` key.
4. Press the `Space` or `Enter` keys and observe the behavior.

**[Expected results]:**
- The dropdown menu is opened.

**[Actual results]:**
- Nothing happens.

**[Notes]:**
- The issue is reproducible for ALL dropdown menus on the page.
- The issue is also reproducible when trying to press the Up/Down arrows.
- Attached a screen recording of the issue.
![dropdowns not accessible via keyboard navigation](https://user-images.githubusercontent.com/18304511/96265820-99073c80-0fce-11eb-89d9-9eba5e4623fc.gif)
